src/amazonqFeatureDev/errors.ts

```diff
@@ -64,6 +70,12 @@ export class ContentLengthError extends ToolkitError {
     }
 }
 
+export class ApiError extends ToolkitError {
+    constructor(message: string, api: string, errorCode: number) {
+        super(message, { code: `${api} - ${errorCode}` })
```

Comment:
@nkomonen-amazon or @dhruvigajjar are there any issues with this code having a space in it (just double checking that it doesn't break anything on the backend)